apps/election-manager/src/lib/votecounting.test.ts

```diff
@@ -754,6 +754,133 @@ test('undervotes counted in n of m contest properly', () => {
   ).toBe(0)
 })
 
+test('overvotes counted in n of m contest properly', () => {
+  // Create mock CVR data
+  const mockCVR: CastVoteRecord = {
+    _ballotId: 'abc',
+    _ballotStyleId: '12D',
+    _ballotType: 'standard',
+    _precinctId: '21',
+    _testBallot: false,
+    _scannerId: '1',
+    'county-commissioners': [
+      'argent',
+      'witherspoonsmithson',
+      'bainbridge',
+      'hennessey',
+    ],
+  }
+
+  // tabulate it
+  let electionTally = computeFullElectionTally(primaryElectionSample, [
+    [mockCVR],
+  ])!
+
+  // The county commissioners race has 4 seats. Each vote less then 4 should be counted
+  // as an additional undervote.
```

Comment:
This comment is about undervotes. Copy-pasta?

Reply:
ah yes my bad